onnxmltools/convert/keras/operator_converters/Pool.py

```diff
@@ -48,10 +49,12 @@ def convert_keras_pooling_core(scope, operator, container, is_global, n_dims,
         else:
             raise RuntimeError("Unsupported padding type '{0}'".format(op.padding))
 
+    output_node_name = scope.get_unique_variable_name('pooling_for_reshape') if is_global else operator.outputs[0].full_name
```

Comment:
This name is not for a node. Maybe change it to `output_tensor_name`? Please also add a comment to the code:
`# The output_tensor_name is used to store the Keras result produced by ONNX operators. For global pooling, a Reshape will be further added to match the actual Keras's output shape.`